ui/raidboss/data/02-arr/trial/ultima-ex.ts

```diff
@@ -0,0 +1,126 @@
+import NetRegexes from '../../../../../resources/netregexes';
+import { Responses } from '../../../../../resources/responses';
+import ZoneId from '../../../../../resources/zone_id';
+import { RaidbossData } from '../../../../../types/data';
+import { TriggerSet } from '../../../../../types/trigger';
+
+export interface Data extends RaidbossData {
+  plasmTargets?: Array<string>;
+  boomCounter: number;
+}
+
+const triggerSet: TriggerSet<Data> = {
+  zoneId: ZoneId.TheMinstrelsBalladUltimasBane,
+  timelineFile: 'ultima-ex.txt',
+  initData: () => {
+    return {
+      plasmTargets: [],
+      boomCounter: 0,
+    };
+  },
+  timelineTriggers: [
+    {
+      // Early Callout for Tank Cleave
+      id: 'Ultima EX Homing Lasers',
+      regex: /Homing Lasers/,
+      beforeSeconds: 4,
```

Comment:
```suggestion
      beforeSeconds: 5,
```

Unless there's a timeline reason, for consistency do 5 seconds before knockback in case somebody wants to arm's length? (if it works)

Reply:
This isn't the knockback, this is the splashing off-tank buster. However, I need to do some additional checking, because one video I saw suggested that this is targeted on anyone not in melee range, not the person who has second threat.

Reply:
Sorry this comment was meant for line 37 below on Vulcan Burst.